src/WebJobs.Script.WebHost/Models/ContainerFunctionExecutionActivityRequest.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
+{
+    public class ContainerFunctionExecutionActivityRequest
+    {
+        public ContainerFunctionExecutionActivityRequest(IEnumerable<ContainerFunctionExecutionActivity> activities)
+        {
+            Activities = activities;
+            FunctionalActivitiesCount = Activities.Count(a => a.IsFunctional());
```

Comment:
It seems the existing payload has all the information needed for the recipient to compute this themselves already, without us having to make any contract changes.

Reply:
True, but for reasons I cannot explain we had 1 CRI (there has been just 1 app where we saw this) where the enums were not serialized correctly across the wire.
https://github.com/Azure/azure-functions-docker-private/issues/175 has sample logs.
So adding an extra int allows the receiver to take action.